deepmd/dpmodel/model/linear_atomic_model.py

```diff
@@ -0,0 +1,288 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+from abc import (
+    abstractmethod,
+)
+from typing import (
+    Dict,
+    List,
+    Optional,
+    Tuple,
+    Union,
+)
+
+import numpy as np
+
+from deepmd.dpmodel import (
+    FittingOutputDef,
+    OutputVariableDef,
+)
+from deepmd.dpmodel.utils.nlist import (
+    build_multiple_neighbor_list,
+    get_multiple_nlist_key,
+    nlist_distinguish_types,
+)
+
+from .base_atomic_model import (
+    BaseAtomicModel,
+)
+from .dp_atomic_model import (
+    DPAtomicModel,
+)
+from .pairtab_atomic_model import (
+    PairTabModel,
+)
+
+
+class LinearAtomicModel(BaseAtomicModel):
+    """Linear model make linear combinations of several existing models.
+
+    Parameters
+    ----------
+    models : list[DPAtomicModel or PairTabModel]
+        A list of models to be combined. PairTabModel must be used together with a DPAtomicModel.
+    """
+
+    def __init__(
+        self,
+        models: List[BaseAtomicModel],
+        **kwargs,
+    ):
+        super().__init__()
+        self.models = models
+        self.distinguish_type_list = [
+            model.distinguish_types() for model in self.models
+        ]
+
+    def distinguish_types(self) -> bool:
+        """If distinguish different types by sorting."""
+        return False
+
+    def get_rcut(self) -> float:
+        """Get the cut-off radius."""
+        return max(self.get_rcuts())
+
+    def get_rcuts(self) -> List[float]:
+        """Get the cut-off radius for each individual models."""
+        return [model.get_rcut() for model in self.models]
+
+    def get_sel(self) -> List[int]:
+        return [model.get_nsel() for model in self.models]
```

Comment:
```suggestion
        return [max([model.get_nsel() for model in self.models])]
```
please check if it is correct.